examples/get-started/first_figure.py

```diff
@@ -121,8 +121,7 @@
 # annotate the latitude and longitude of the region.
 #
 # The ``frame`` parameter is used to add a frame to the figure. For now, it
-# will be set to ``True`` to use default settings, but later tutorials will
-# show how ``frame`` can be used to customize the axes, gridlines, and titles.
+# will be set to ``"a"`` to use automatic annotation settings.
```

Comment:
I want to prevent users from thinking that 'a' stands for automatic (the 'a' refers to 'annotate'). How about a sentence like this?
```suggestion
# will be set to ``"a"`` to **a**nnotate the axes automatically.
```